src/main/resources/static/js/testrules.js

```diff
@@ -16,7 +16,7 @@ jQuery(document).ready(function () {
             var context = ko.contextFor(event.target);
             self.rulesBindingList.splice(context.$index(), 1);
             if (self.rulesBindingList().length == 0) {
-                logMessages("Deleted all rule types, but we need atleast one Rule type, Here add default rule type");
+                logMessage("Deleted all rule types, but we need atleast one Rule type, Here add default rule type");
```

Comment:
Forgot space between 'at least'. And the entire sentence reads a bit confusing ... Maybe something like: "All rule types were deleted, but we need at least one so we add a default rule here." would be better?

Reply:
Done.